packages/core/admin/admin/src/content-manager/components/DynamicTable/BulkActionsBar/index.js

```diff
@@ -214,34 +214,46 @@ const BulkActionsBar = ({
   };
 
   const handleBulkDelete = async () => {
+    let result = null;
+
     try {
       setIsConfirmButtonLoading(true);
-      await onConfirmDeleteAll(selectedEntries);
+      result = await onConfirmDeleteAll(selectedEntries);
```

Comment:
This way to solve the issue looks good to me 👍 

Since it adds a bit a complexity to these 3 functions, and they're really similar, I'm wondering if we couldn't make a single handleBulkAction fn and pass the callbacks as params? like:

```js
const handleBulkAction = (confirmAction, toggleModal) => {
 // manage try, catch, result etc in there
}
```

Reply:
Yup! updated.